doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -141,7 +142,7 @@ It is based on proven, standards-based networking technologies that today suppor
 
 Based on Network Layering concept that introduced ealier , Tungsten Fabric Controller , can be considered as SDNu and SDNo
 
-The SDN controlelr can exposed an NB API that can be consumbed by ETSI MANO for VNF/CNF onboarding , Network Service onboarding and dynaimc Service chaining 
+The SDN controller can exposes a NB API that can be consumed by ETSI MANO for VNF/CNF onboarding, network service onboarding and dynamic service function chaining.
```

Comment:
@ASawwaf Can you please delete "can" from "The SDN controller **can** exposes a NB .."

Reply:
@pgoyal01 done removed

Reply:
@ASawwaf Thanks